test/jdk/java/nio/channels/spi/SelectorProvider/inheritedChannel/java.policy.fail

```diff
@@ -1,11 +1,13 @@
 //
 //
-// 
+//
```

Comment:
Are the empty lines significant?

Reply:
No. I've tidied that up. Was there before